### PR TITLE
refactor: add new alert formats for all alerts

### PR DESCRIPTION
**Pull request by HugoIr**
### Describe your changes
Add new alert formats for all alerts

### Acceptance criteria
- User understand the alert message clearly

### Checklist
- [x] I have performed a self-review of my code
- [x] Code successfully run on local
- [x] Feature / changes tested on local
- [x] No failing unit test
- [ ] Passed UAT Test
- [x] Sonarqube tested 
- [ ] Required any changes to environment variable

### CODE REVIEW

File: alerts/management/commands/run_cron_alerts.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import os
+from tracemalloc import start
```
Comment (andrew-susanto) [inline alerts/management/commands/run_cron_alerts.py:2]:
```suggestion

```

File: alerts/management/commands/run_cron_alerts.py [hunk 2]
```diff
@@ -7,7 +8,7 @@
 
 from django.core.mail import get_connection
 from django.template.loader import render_to_string
-from django.utils import timezone
+from django.utils import timezone, dateformat
```
Comment (andrew-susanto) [inline alerts/management/commands/run_cron_alerts.py:11]:
```suggestion
from django.utils import timezone
```

File: alerts/management/commands/run_cron_alerts.py [hunk 3]
```diff
@@ -37,11 +39,12 @@ def get_success_rate(self, monitor):
         }
 
         alerts_config, _ = AlertsConfiguration.objects.get_or_create(user=monitor.user)
-        time_theshold = timezone.now() - timedelta(seconds=time_window_in_seconds[alerts_config.time_window])
+        end_time = timezone.now()
+        start_time = timezone.now() - timedelta(seconds=time_window_in_seconds[alerts_config.time_window])
```
Comment (andrew-susanto) [inline alerts/management/commands/run_cron_alerts.py:43]:
```suggestion
        start_time = end_time - timedelta(seconds=time_window_in_seconds[alerts_config.time_window])
```

File: alerts/management/commands/run_cron_alerts.py [hunk 4]
```diff
@@ -51,7 +54,7 @@ def get_success_rate(self, monitor):
         success_rate = 100
         if success_count['total'] != 0:
             success_rate = success_count['s'] / (success_count['total']) * 100
-        return success_rate
+        return success_rate, start_time.strftime("%d %b %Y, %H:%M:%S"), end_time.strftime("%d %b %Y, %H:%M:%S")
```
Comment (andrew-susanto) [inline alerts/management/commands/run_cron_alerts.py:57]: ini dibikin variabel dulu aja hug
```suggestion
        formatted_success_rate = round(float(success_rate), 2)
        formatted_start_time = start_time.strftime("%d %b %Y, %H:%M:%S")
        formatted_end_time = end_time.strftime("%d %b %Y, %H:%M:%S")
        return formatted_success_rate , formatted_start_time, formatted_end_time
```

File: alerts/management/commands/run_cron_alerts.py [hunk 5]
```diff
@@ -174,16 +180,18 @@ def worker(self, type):
                 print(f"[{timezone.now()}] Send {type} alert for monitor id:{monitor_id}")
                 
                 monitor = APIMonitor.objects.get(id=monitor_id)
+                success_rate, start_time, end_time = self.get_success_rate(monitor)
+                success_rate = round(float(success_rate), 2)
```
Comment (andrew-susanto) [inline alerts/management/commands/run_cron_alerts.py:184]: skalian di round di function get_success_rate aja hug

**Review by andrew-susanto — APPROVED** [review]

**Review by luthfifahlevi — APPROVED** [review]
Nice, thank you

**Pull request merged**